src/index.js

```diff
@@ -38,3 +37,19 @@ export const renderReactWithAphrodite = (name, component) => hypernova({
     return component;
   },
 });
+
+export const renderReactWithAphroditeStatic = (name, component) => hypernova({
+  server() {
+    return (props) => {
+      const { html, css } = StyleSheetServer.renderStatic(() => (
+        ReactDOMServer.renderToStaticMarkup(React.createElement(component, props))
+      ));
+
+      const style = `<style data-aphrodite>${css.content}</style>`;
+
+      return `${style}\n${html}`;
```

Comment:
Thanks for commenting Josh. Why do we need the classes from aphrodite in the output string? Based on the non-static rendered, it seems like those are only used so that the client can be initialized properly https://github.com/airbnb/hypernova-aphrodite/blob/fece1bbf9d689d86a67f915d469c7e7b05b247ec/src/index.js#L24-L25
which isn't necessary for the static-renderer. cc @lencioni